.github/workflows/crawler.yml

```diff
@@ -21,13 +21,20 @@ jobs:
           ruby-version: 2.7
           bundler-cache: true
 
-      - run: bundle exec ruby crawler.rb ${{ inputs.args }}
+      - run: |
+          rm -f index.yaml index.zip index-new.yaml index-new.zip
```

Comment:
Up to you.

To be on the same page it's not a problem to keep it here in `relaton/support` because in case we will need to remove it it's just one place.

So better put a temporary solution in `crawler.yml` than in each `crawler.rb`